content/influxdb/v2.0/write-data/_index.md

```diff
@@ -163,15 +166,41 @@ If you missed the Quick Start option, you can [manually create a scraper](/influ
 that scrapes data from the `/metrics` endpoint.
 {{% /note %}}
 
-
 ---
 
-## Other ways to write data
+## Load data from sources in the InfluxDB user interface (UI)
+
+Load data from the following sources in the InfluxDB user interface (UI):
 
-There are multiple options for writing data into InfluxDB, including both no-code and developer solutions.
+- [Client libraries](#load-data-from-a-client-library-in-the-ui)
+- [Telegraf plugins](#load-data-from-a-telegraf-plugin-in-the-ui)
 
-  - [No-code solutions](/influxdb/v2.0/write-data/no-code)
-  - [Developer tools](/influxdb/v2.0/write-data/developer-tools)
+### Load data from a client library in the UI
+
+1.  In the navigation menu on the left, click **Data (Load Data)** > **Sources**.
+    {{< nav-icon "data" >}}
+2. Do one of the following:
+   - Enter a specific client library to search for in the **Search data writing methods** field.
+   - Scroll down to browse available client libraries.
+3. Click the client library to load data from.
+4. Under **Code Sample Options**, you'll see a list of your InfluxDB [tokens](/influxdb/v2.0/reference/glossary/#token) and [buckets](/influxdb/v2.0/reference/glossary/#bucket). Select both an authentication token and a bucket to write your data to. The selected token and bucket are automatically added to scripts on the page that you can use to initialize a client and write data.
+5. Click the **Copy to Clipboard** buttons under a script to easily paste the script into your terminal or save the script to reuse for automation.
+6. Run the scripts on the page to do the following as needed:
+   - Install the package, libraries, or client
+   - Write data
+   - Execute a Flux query
+
+### Load data from a Telegraf plugin in the UI
+
+1. In the navigation menu on the left, click **Data (Load Data)** > **Sources**.
+    {{< nav-icon "data" >}}
+2. Do one of the following:
+   - Enter a specific Telegraf plugin to search for in the **Search data writing methods** field.
+   - Scroll down to browse available plugins.
+3. Click the plugin to load data from.
+4. [Install Telegraf](/telegraf/v1.15/introduction/installation/).
+5. Copy the default configuration script in the UI, and then add the script to your [Telegraf configurations](/influxdb/v2.0/telegraf-configs/).
```

Comment:
This should probably link to information about the Telegraf configuration file (https://docs.influxdata.com/telegraf/v1.15/introduction/getting-started/#configure-telegraf). We could then add an optional step about adding the Telegraf configuration to InfluxDB (this is what /influxdb/v2.0/telegraf-configs/ walks through).